py/tracks/latest.toml

```diff
@@ -13,6 +13,7 @@ fixtures = [
 queries = [
     "../specs/group_by.toml",
     "../specs/like.toml",
+    "../specs/regex.toml",
```

Comment:
the files listed under `queries` will use setup/teardown from the file listed under `fixtures`. So in order for this to work correctly the setup/teardown of `regex.toml` must match the one of `group_by.toml`

You can either move `regex.toml` into `full` or adapt the setup so that it matches.

Not sure about the analyzer stuff - why is that needed?